tools/scripts/firehose.py

```diff
@@ -239,7 +240,7 @@ def generate_events(events, job, time_delta):
         num_events = events
 
     for i in range(num_procs):
-        p = multiprocessing.Process(target=firehose, args=(job, num_events, created_stamp, modified_stamp))
+        p = multiprocessing.Process(target=firehose, args=(job, num_events, created_stamp, modified_stamp, job_created))
```

Comment:
@kdelee ^ firehose must have gotten pretty slow after we did partitioning? Because it wasn't leveraging partitions.

Reply:
@kdelee do you have something else now for testing event payloads on scale?